### DISCUSSION TITLE
How to debug "stuck" cron job (small scale deployment)

### DISCUSSION THREAD

**Discussion by seybsen** [pretix/pretix#2000] [Q&A]
Hi,

somehow the [cronjob](https://docs.pretix.eu/en/latest/admin/installation/docker_smallscale.html#cronjob) of our pretix 3.16 small scale docker deployment seems to get stuck and kills all docker communication with the container.
Thus all commands like `docker logs pretix`, `docker stats pretix`, `docker inspect pretix`, etc. do not work until the complete docker service is restarted.

Any tips on debugging this issue would be much appreciated :slightly_smiling_face:

**Comment by raphaelm** [comment]
Wow, this is weird, and I haven't seen this before. Are any system resources (CPU, memory) under full load when this happens?

**Reply by seybsen** [reply]
All ressources seem to be on a normal level with some headroom.

Pretix works just fine even after docker container gets unresponsive. I just can't control the container anymore.
This sounds more like an docker (Docker version 20.10.5, build 55c4c88) issue for me tho.

Anyways, it would be interesting to see what pretix is doing at the moment this is happening. Do you have any tips where to dig into the runperiodic job? Especially how to increase verbosity: https://github.com/pretix/pretix/blob/002ab4aa06fbdbbdaf0a3f5e34f889aea0f21e69/src/pretix/base/management/commands/runperiodic.py#L23

**Reply by raphaelm** [reply]
Yup, really sounds like a docker issue.

You can increase verbosity with the ``-v 2`` flag, which should let you know which jobs it is running
